### PR TITLE
Add new AR docs for web

### PR DESCRIPTION
**Pull request by HedwigAR**
This PR adds a page with some explanation about upcoming web versions of the native AR widgets. I've currently put it next to existing AR documentation (in mobile/using-mobile-capabilities/augmented-reality), but I'm totally open to putting it elsewhere. I couldn't think of a better place to put that.

### CODE REVIEW

File: content/en/docs/refguide/mobile/using-mobile-capabilities/augmented-reality/how-to-web-xr.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+{{< figure src="/attachments/howto/mobile/native-mobile/ar-parent/how-to-webxr/cube-no-webxr-button.jpg" alt="How the page appears without WebXR support"   width="400"  >}}
+
+Here is that same scene with the button:
+
+{{< figure src="/attachments/howto/mobile/native-mobile/ar-parent/how-to-webxr/cube-webxr-button.jpg" alt="How the page appears with WebXR support"   width="400"  >}}
+
+It is not possible to start the experience without the user clicking the button. This is due to the user needing to give permission to have their space taken over by this experience, and not being surprised by this happening.
+
+Depending on which browser you plan to use, you will need to enable the WebXR feature flags.
```
Comment (ConnorLand) [inline content/en/docs/refguide/mobile/using-mobile-capabilities/augmented-reality/how-to-web-xr.md]: Should we list those flags here?

Comment (HedwigAR) [reply content/en/docs/refguide/mobile/using-mobile-capabilities/augmented-reality/how-to-web-xr.md]: Yeah, great point. I'll add that

File: content/en/docs/refguide/mobile/using-mobile-capabilities/augmented-reality/how-to-web-xr.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+#### 4.1.2 Material Tab
+
+* **Type**
+  * **Object** – Only available for **3D Object (XR)**. Selecting this will make the object use the material as present on the 3D Object.
+  * **Texture** – Sets a static or dynamic image for this object. This image will then be mapped onto the 3D object.
+  * **Color** – Sets a solid color on an object. This should be provided in the #rrggbb format.
+* **Opacity** – The transparency of the object. 0 is transparent, while 1 is opaque.
+
+* **Lighting**
+  * **Type** – The kind of lighting that should be used on this object. **Simple** lighting is more basic looking, but easier to render, leading to better framerates. **Realistic** lighting shows more complex reflections and adds options for the type of material this object is supposed to have:
```
Comment (ConnorLand) [inline content/en/docs/refguide/mobile/using-mobile-capabilities/augmented-reality/how-to-web-xr.md:124]: I see the Realistic lighting property is documented in the General 3.1.1 section. Does **Simple** lighting prop need to be defined too? I don't see where it's documented.

**Comment by ConnorLand** [comment]
Looks mostly done to me. After @takumagott weighs in with any change suggestions, then I'll merge.

File: content/en/docs/refguide/mobile/using-mobile-capabilities/augmented-reality/how-to-web-xr.md (hunk 2 above)
Comment (ConnorLand) [reply content/en/docs/refguide/mobile/using-mobile-capabilities/augmented-reality/how-to-web-xr.md:124]: @HedwigAR just dropping your name if that helps visibility.

Comment (HedwigAR) [reply content/en/docs/refguide/mobile/using-mobile-capabilities/augmented-reality/how-to-web-xr.md:124]: That prop isn't available here, you set the lighting on each object, I've reworded the section and added a prop that came in after I wrote this.

**Comment by takumagott** [comment]
Looks good to me 👍

**Review by takumagott — APPROVED** [review]

**Comment by ConnorLand** [comment]
Putting release date label on to remind me to add the Marketplace component URL before merging.

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**